snapshot/git/gitdb/bundlestore.go

```diff
@@ -96,6 +96,14 @@ func (b *bundlestoreBackend) uploadLocalSnapshot(s *localSnapshot, db *DB) (sn s
 
 			mergeBase, err := db.dataRepo.RunSha("merge-base", streamHead, commitSha)
 
+			if mergeBase == commitSha {
```

Comment:
Commented.

I just tried to test that master worked passed, and it error'ed. Whoops!